conf/messages

```diff
@@ -115,13 +113,6 @@ honestyDeclaration.lossOfStaff = of loss of staff essential to the VAT process
 agent.honestyDeclaration.lossOfStaff = my client was affected by loss of staff essential to the VAT process
 honestyDeclaration.other =
 
-# Errors
-# ----------------------------------------------------------
-error.prefix = Error:
-# Not sure if this is as per design
-error.clickAcceptAndContinue = Please click ’Accept and continue’
-error.clickContinue = Please click ’Continue’
```

Comment:
Arguably you can get rid of these two, but you'd also have to remove the form validation behind it (the form validation is quite redundant tbh)

Reply:
ignore